packages/storefront/src/pages/utilities/scss/functions.md

```diff
@@ -90,12 +90,12 @@ p {
 }
 ```
 
-### Headline
+### Heading
 
 Given values are:  
-`title-large | headline-1 | headline-2 | headline-3 | headline-4 | headline-5`
+`title-large | heading-1 | heading-2 | heading-3 | heading-4 | heading-5`
```

Comment:
Do these utils even exist??

Reply:
hehe, we'll have to update this section in general in the upcoming days since this package and docs are deprecated.